Plugins/IMA/IMAPlugin.swift

```diff
@@ -312,6 +312,7 @@ enum IMAState: Int, StateProtocol {
         case .STREAM_LOADED: self.notify(event: AdEvent.AdStreamLoaded())
         case .TAPPED: self.notify(event: AdEvent.AdTapped())
         case .THIRD_QUARTILE: self.notify(event: AdEvent.AdThirdQuartile())
+        default: break 
```

Comment:
@gal-orlanczyk  we will remove all cases that handle dynamic ads since it's not supported and will log it by default.

Thanks!